proposals/community-strategy/social-media.md

```diff
@@ -0,0 +1,41 @@
+# Social media
+
+## Youtube
+
+[Youtube](https://www.youtube.com/) is an important resource for development. 
+Especially when it comes to modern web frameworks, such as React, VueJS or Angular. Some development Youtube channels 
+reach some millions of developers.
+
+### Some Youtube channels in the relevant area
+
+#### Embedded
+
+- [Andreas Spiess](https://www.youtube.com/channel/UCu7_D0o48KbfhpEohoP7YSQ)
+- [GreatScott!](https://www.youtube.com/user/greatscottlab)
+- [Make:](https://www.youtube.com/c/MAKE/featured)
+
```

Comment:
I had this idea for reaching the most known YouTubers and ask for feedback /videos about W3C too. However, I think we might need to improve more on node-wot/other runtimes before going down this path. Once we reach something that is "almost" production-ready, YouTubers are the fastest means to connect with a larger audience.

Reply:
I'm in contact with Andreas Spiess. He is fine with covering these kind of topics. There is just one very important thing for him: He needs to get these things done within one or two days. Otherwise he will drop it, because his audience also demands easy to use solution. He said that a lot of professionals in his audience are into fast prototyping. Their goal is to test a lot of products in a short period of time, in order to decide what they want to get forward with. So, I agree with you. That is the biggest issue for now. That is why I would skip this for now and come back to the Youtube topic later. Reaching open source communities and getting implemented should be the main goal for now.